packages/webdriverio/src/middlewares.ts

```diff
@@ -38,13 +35,19 @@ export const elementErrorHandler = (fn: Function) => (commandName: string, comma
 
                 return result
             } catch (err: any) {
+                if (err.name === 'element not interactable') {
```

Comment:
I believe in Safari the error name is different which is why the CI test is failing, mind giving it a try with Safari?

Reply:
I took the stacktrace from another error which had the session info. Since I added the tests without having the fix from the other PR in this I made a guess and seems like this doesn't have the session info when running Safari so I it is easily solved 👍
Will fix it and when green merge it